universal/sleep.md

```diff
@@ -170,7 +170,7 @@ This guide is primarily for dGPU but works the exact same way with NVMe drives(a
 
 To verify you have working CPU Power Management, see the [Fixing Power Management](../universal/pm.md) page. And if not, then patch accordingly.
 
-Also note that incorrect frequency vectors can result in wake issues, so either verify you're using the correct SMBIOS or adjust the frequency vectors of your current SMBIOS with CPUFriend. Tools like [one-key-cpufriend](https://github.com/stevezhengshiqi/one-key-cpufriend) are known for creating bad frequency vectors so be careful with tools not used by Dortania.
+Also note that incorrect frequency vectors can result in wake issues, so verify you're using the correct SMBIOS.
```

Comment:
I am not sure whether wake issues are caused exactly by FV, better to just mention <power management data>.